pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -375,6 +375,18 @@ func (c *HAProxy) PMMContainer(spec *api.PMMSpec, secret *corev1.Secret, cr *api
 			},
 		}
 		ct.Env = append(ct.Env, sidecarEnvs...)
+
+		fvar := true
+		ct.EnvFrom = []corev1.EnvFromSource{
```

Comment:
Should we have the same for PXC and ProxySQL?

Reply:
We already have it for ProxySQL: https://github.com/percona/percona-xtradb-cluster-operator/blob/f3c54824d3144a20d2f07c258e6e1b56c5f73bdc/pkg/pxc/app/statefulset/proxysql.go#L343-L352 and for PXC: https://github.com/percona/percona-xtradb-cluster-operator/blob/f3c54824d3144a20d2f07c258e6e1b56c5f73bdc/pkg/pxc/app/statefulset/node.go#L430-L439

It was missing in PMM container for HAProxy, so I added it